dandiapi/api/views/auth.py

```diff
@@ -124,7 +124,9 @@ def user_questionnaire_form_view(request: HttpRequest) -> HttpResponse:
             not questionnaire_already_filled_out
             and user_metadata.status == UserMetadata.Status.INCOMPLETE
         ):
-            is_edu_email: bool = user.email.endswith('.edu')
+            is_edu_email: bool = user.email.endswith('.edu') or user.email.endswith(
+                '@alleninstitute.org'
+            )
```

Comment:
I would just suggest changing this variable name to reflect it's new meaning -

```suggestion
            should_auto_approve: bool = user.email.endswith('.edu') or user.email.endswith(
                '@alleninstitute.org'
            )
```

Reply:
done